index.js

```diff
@@ -17,5 +18,6 @@ module.exports = {
         assertFunctionNames: ['expect', 'expectSaga'],
       },
     ],
+    'testing-library/no-debug': 'warn',
```

Comment:
Looks like there is also a[`plugin:testing-library/react` config](https://github.com/testing-library/eslint-plugin-testing-library/blob/4727c63df65bd4ee2b95e5d9db02316678d64a11/lib/index.ts#L61-L67) which includes all the recommended rules but also includes this one (and `testing-library/no-dom-import`). Might be worth using it instead of `plugin:testing-library/recommended`?

Reply:
(extremely belated) Thanks. Good call.

I hope you and the kids are well. 😄